.spelling

```diff
@@ -0,0 +1,724 @@
+Get-CimInstance
+New-CimInstance
+Remove-CimInstance
+Set-CimInstance
+Invoke-CimMethod
+Get-CimSession
+New-CimSession
+Remove-CimSession
+New-CimSessionOption
+Export-Clixml
+Import-Clixml
+Get-ComputerInfo
+ConvertFrom-Csv
+ConvertTo-Csv
+Export-Csv
+Import-Csv
+Find-DscResource
+Register-EngineEvent
+Unregister-Event
+Get-EventSubscriber
+Get-ExecutionPolicy
+Set-ExecutionPolicy
+New-FileCatalog
+Test-FileCatalog
+Get-FileHash
+Export-FormatData
+Get-FormatData
+Update-FormatData
+New-Guid
+Get-InstalledModule
+Get-InstalledScript
+Clear-ItemProperty
+Copy-ItemProperty
+Get-ItemProperty
+Move-ItemProperty
+New-ItemProperty
+Remove-ItemProperty
+Rename-ItemProperty
+Set-ItemProperty
+Get-ItemPropertyValue
+ConvertTo-Json
+Get-LocalGroup
+New-LocalGroup
+Remove-LocalGroup
+Rename-LocalGroup
+Set-LocalGroup
+Add-LocalGroupMember
+Get-LocalGroupMember
+Remove-LocalGroupMember
+Import-LocalizedData
+Disable-LocalUser
+Enable-LocalUser
+Get-LocalUser
+New-LocalUser
+Remove-LocalUser
+Rename-LocalUser
+Set-LocalUser
+Get-LogProperties
+Set-LogProperties
+New-ModuleManifest
+Test-ModuleManifest
+Update-ModuleManifest
+Export-ModuleMember
+Register-ObjectEvent
+Find-PackageProvider
+Get-PackageProvider
+Import-PackageProvider
+Install-PackageProvider
+Get-PackageSource
+Register-PackageSource
+Set-PackageSource
+Unregister-PackageSource
+Import-PowerShellDataFile
+Disable-PSBreakpoint
+Enable-PSBreakpoint
+Get-PSBreakpoint
+Remove-PSBreakpoint
+Get-PSCallStack
+Set-PSDebug
+Get-PSDrive
+New-PSDrive
+Remove-PSDrive
+Enter-PSHostProcess
+Exit-PSHostProcess
+Get-PSHostProcessInfo
+Get-PSProvider
+Get-PSReadlineKeyHandler
+Remove-PSReadlineKeyHandler
+Set-PSReadlineKeyHandler
+Get-PSReadlineOption
+Set-PSReadlineOption
+Get-PSRepository
+Register-PSRepository
+Set-PSRepository
+Unregister-PSRepository
+New-PSRoleCapabilityFile
+Connect-PSSession
+Disconnect-PSSession
+Exit-PSSession
+Get-PSSession
+Receive-PSSession
+Remove-PSSession
+Get-PSSessionCapability
+Disable-PSSessionConfiguration
+Enable-PSSessionConfiguration
+Get-PSSessionConfiguration
+Register-PSSessionConfiguration
+Set-PSSessionConfiguration
+Unregister-PSSessionConfiguration
+New-PSSessionConfigurationFile
+Test-PSSessionConfigurationFile
+New-PSSessionOption
+Disable-PSTrace
+Enable-PSTrace
+New-PSTransportOption
+Disable-PSWSManCombinedTrace
+Enable-PSWSManCombinedTrace
+Invoke-RestMethod
+Find-RoleCapability
+Debug-Runspace
+Get-Runspace
+Disable-RunspaceDebug
+Enable-RunspaceDebug
+Get-RunspaceDebug
+New-ScriptFileInfo
+Test-ScriptFileInfo
+Update-ScriptFileInfo
+ConvertFrom-SddlString
+ConvertFrom-SecureString
+ConvertTo-SecureString
+Set-StrictMode
+ConvertFrom-StringData
+New-TemporaryFile
+Get-TimeZone
+Set-TimeZone
+Get-TypeData
+Remove-TypeData
+Update-TypeData
+Get-UICulture
+Get-WinEvent
+New-WinEvent
+Connect-WSMan
+Disconnect-WSMan
+Test-WSMan
+Invoke-WSManAction
+Disable-WSManCredSSP
+Enable-WSManCredSSP
+Get-WSManCredSSP
+Get-WSManInstance
+New-WSManInstance
```

Comment:
White-listing all possible cmdlet names is tedious. Maybe we can just put all usages of them in codesnippet escaping with backticks? Does it stops spellchecker?